tests/agent_features/test_configuration.py

```diff
@@ -581,6 +583,7 @@ def test_translate_deprecated_ignored_params_with_new_setting():
         ("otlp_port", 0),
     ),
 )
+@override_generic_settings(global_settings(), {"host": None})
```

Comment:
This hasn't been addressed either. Since this isn't really related to the current PR and metrics at all, I think we should just drop this change from this PR. There's plenty of other changes going on here without adding this in.